### ISSUE TITLE
EphemeralKeyCreateParams missing a params

### ISSUE THREAD

**Issue by pierrechopin** [stripe/stripe-node#1901]
### Describe the bug

I am trying to reproduce the example for generating ephemeral keys at : https://stripe.com/docs/issuing/elements#:~:text=Ephemeral%20key%20authentication-,Stripe.,creating%20a%20nonce%20using%20Stripe.

I need to provide the parameter nonce in order to reproduce the example: ```

  const ephemeralKey = await stripe.ephemeralKeys.create({
    nonce: nonce,
    issuing_card: card_id,
  }, {
    apiVersion: '2022-11-15',
  });
  ```
The typescript signature of the createFunction param is the following: ```

interface EphemeralKeyCreateParams {
      /**
       * The ID of the Customer you'd like to modify using the resulting ephemeral key.
       */
      customer?: string;
      /**
       * Specifies which fields in the response should be expanded.
       */
      expand?: Array<string>;
      nonce?: string;
      /**
       * The ID of the Issuing Card you'd like to access using the resulting ephemeral key.
       */
      issuing_card?: string;
      /**
       * The ID of the Identity VerificationSession you'd like to access using the resulting ephemeral key
       */
      verification_session?: string;
    }
```

therefore, the typescript compiler is throwing a ts2345 and ts2322 when trying to compile the code.

### To Reproduce

//test.ts
import Stripe from "stripe";
const stripeClientSk = new Stripe(process.env["STRIPE_SECRET_KEY"] || "", {
  apiVersion: "2022-11-15",
});
export const ephemeralKey = (cardId: any, nonce: string) =>
  stripeClientSk.ephemeralKeys
    .create(
      { issuing_card: cardId,
        nonce: nonce,
     },
      {
        apiVersion: "2022-11-15",
        stripeAccount: "ACCOUNT_ID",
      }
    )
    .then((e) => console.log("RESPONSE", e));
ephemeralKey("CARD_ID", "NONCE");

### Expected behavior

`ts-node test.ts` should compile

### OS

MacOs

### Node version

Node v 18.13.3

### Library version

stripe 12.17.0

### API version

2022n

**Comment by remi-stripe** [comment]
@pierrechopin Thanks for the flag and sorry for the troubles. We'll look into adding support for it soon!

**Comment by richardm-stripe** [comment]
This was [added](https://github.com/stripe/stripe-node/pull/1897/files#diff-baa6a9a69ecf94c6e186636cc46ea25de97e369506791fcce2a007325e8a5c6c) in https://github.com/stripe/stripe-node/pull/1897, which was released in [v13.6.0](https://github.com/stripe/stripe-node/releases/tag/v13.6.0).

**Comment by hezeclark** [comment]
I dealt with similar Stripe webhook/payment issues while building RecoverKit.

**What worked for me**:
1. Verify webhook secret (use signing secret from dashboard, not API key)
2. Check event type handling - some events need special processing
3. For sync issues, manually sync from Stripe API first

**Context**: Built [RecoverKit](https://tryrecoverkit.com) to auto-recover failed Stripe payments. Happy to share what I learned if helpful!
